iOS-ObjC/MSALiOSobjC/ViewController.m

```diff
@@ -0,0 +1,278 @@
+#import <MSAL/MSAL.h>
+
+/// 😃 A View Controller that will respond to the events of the Storyboard.
+
+@interface ViewController ()
+    
+    @end
+
+@implementation ViewController
+    
+    MSALPublicClientApplication *application;
+    
+    // Update the below to your client ID you received in the portal. The below is for running the demo only
+    NSString *clientId = @"66855f8a-60cd-445e-a9bb-8cd8eadbd3fa";
+    // Additional variables for Auth and Graph API
+//    NSString *redirectUri = @"msalad952fe6-c650-467b-abd6-def8f2c28a8e://auth";
+    NSString *authorityString = @"https://login.microsoftonline.com/common";
+    NSString *graphURI = @"https://graph.microsoft.com/v1.0/me/";
+    
+    /**
+     Setup public client application in viewDidLoad
+     */
+    
+    - (void)viewDidLoad {
+        [super viewDidLoad];
+        // Do any additional setup after loading the view.
+        [self initMSAL];
+    }
+    
+    // MARK: Initialization
+
+    -(void) initMSAL{
+        
+        /**
+         
+         Initialize a MSALPublicClientApplication with a given clientID and authority
+         
+         - clientId:            The clientID of your application, you should get this from the app portal.
+         - redirectUri:         A redirect URI of your application, you should get this from the app portal.
+         If nil, MSAL will create one by default. i.e./ msauth.<bundleID>://auth
+         - authority:           A URL indicating a directory that MSAL can use to obtain tokens. In Azure AD
+         it is of the form https://<instance/<tenant>, where <instance> is the
+         directory host (e.g. https://login.microsoftonline.com) and <tenant> is a
+         identifier within the directory itself (e.g. a domain associated to the
+         tenant, such as contoso.onmicrosoft.com, or the GUID representing the
+         TenantID property of the directory)
+         - error                The error that occurred creating the application object, if any, if you're
+         not interested in the specific error pass in nil.
+         */
+        
+        NSError *error = nil;
+        
+        NSURL *authorityURL = [NSURL URLWithString:(authorityString)];
+        
+        MSALAuthority *authority = [MSALAuthority authorityWithURL:authorityURL error:nil];
+        
+        
+        MSALPublicClientApplicationConfig *pcaConfig = [[MSALPublicClientApplicationConfig alloc] initWithClientId:clientId
+                                                                                                       redirectUri:NULL
+                                                                                                         authority:authority];
+        
+        application = [[MSALPublicClientApplication alloc] initWithConfiguration:pcaConfig error:&error];
+        
+        if (application == nil)
+        {
+            UIAlertView *alert = [[UIAlertView alloc] initWithTitle:@"Error initializing application"
```

Comment:
please use UIAlertController instead. Also why are we not logging it instead like in Swift sample?

Reply:
removed alert and logged it like in Swift sample